appveyor.yml

```diff
@@ -1,5 +1,5 @@
 environment:
-  nodejs_version: "0.10"
+  nodejs_version: "6"
```

Comment:
How intentional was it that this was building against node 0.10? I'd guess that the latest LTS is a reasonable minimum target to support. With a bit more work with "any-promise" I bet I _could_ get us building on 0.10 though.